src/table/use-sticky-scrollbar.ts

```diff
@@ -90,7 +90,10 @@ export function useStickyScrollbar(
   // containing block, if present, is below the app layout and above the overflow
   // parent, which is a pretty safe assumption.
   const [hasContainingBlock, setHasContainingBlock] = useState(false);
-  const consideredFooterHeight = hasContainingBlock ? 0 : footerHeight;
+  // We don't take into account footer height when the scroll parent is child of document body.
```

Comment:
Just to make sure I understand:
When the scrolling parent is a direct child of document.body, we assume the footer is outside the overflow parent and don't need to consider its height (we do `consideredFooterHeight=0`). Is that it? 😃

Reply:
Not necessarily direct child, rest is yes.